mne/source_estimate.py

```diff
@@ -1374,18 +1374,14 @@ def plot(self, subject=None, surface='inflated', hemi='lh',
         hemi : str, 'lh' | 'rh' | 'split' | 'both'
             The hemisphere to display. Using 'both' or 'split' requires
             PySurfer version 0.4 or above.
-        colormap : str
-            The type of colormap to use.
+        colormap : str | array
+            Name of colormap to use or a custom look up table. If array, must
+            be (n x 3) or (n x 4) array for with RGB or RGBA values between 0
+            and 255.
```

Comment:
@mainakjas, I filled this out a little more. n can be between 2 and a long int. I don't think it's adding more than this to the docstring though because it takes some explanation and will just weigh everything down. If someone is passing custom colormaps, I assume they either know what they're doing or are willing to go through some trial and error.